Cargo.toml

```diff
@@ -21,6 +21,7 @@ categories = ["data-structures"]
 serde = { version = "1.0.123", optional = true }
 tinystr-macros = { version = "0.2", path = "./macros" }
 tinystr-raw = { version = "0.1.1", path = "./raw" }
+zerovec = {version = "0.2.0", optional = true }
 
```

Comment:
Suggestion: Add docs to the tinystr crate so that people know this feature exists without reading the code